volatility/framework/caching.py

```diff
@@ -0,0 +1,9 @@
+import functools
+import volatility
+
+
+def lru_cache(*args, **kwargs):
+    if not volatility.CACHING:
+        return lambda x: x
+    raise Exception
```

Comment:
I must be missing something... why is this exception here?

Reply:
That's entirely my bad, it's so I could determine when the wrapper was being applied and make sure the caching value had been set correctly (setting it to late would cause some things to have been wrapped by it early).  Good spot, thanks!  5:D

Reply:
Works now, thanks.